tests/countries/test_falkland_islands.py

```
@@ -0,0 +1,199 @@
+    def setUpClass(cls):
+        years = range(1983, 2050)
+        super().setUpClass(FalklandIslands, years=years, years_non_observed=years)
+        cls.government_holidays = FalklandIslands(categories=GOVERNMENT, years=years)
+
+    def test_country_aliases(self):
+        self.assertAliases(FalklandIslands, FK, FLK)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(FalklandIslands(years=1982))
+
+    def test_special_holidays(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)